src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -280,24 +307,26 @@ def test_upper_bound_ncolour_intergrid(dist_mem):
     if dist_mem:
         assert loops[1]._upper_bound_name == "colour_halo"
         assert (loops[1]._upper_bound_fortran() ==
-                "last_cell_all_colours_field1(colour, 1)")
+                "last_halo_cell_all_colours_field1(colour, 1)")
         # We can't apply redundant computation to increase the depth of the
         # access to the halo as it is not supported for inter-grid kernels.
         # Therefore we manually unset the upper bound halo depth to indicate
         # that we access the full depth.
         loops[1]._upper_bound_halo_depth = None
         assert (loops[1]._upper_bound_fortran() ==
-                "last_cell_all_colours_field1(colour, "
+                "last_halo_cell_all_colours_field1(colour, "
                 "max_halo_depth_mesh_field1)")
     else:
         assert loops[1]._upper_bound_name == "ncolour"
         assert (loops[1]._upper_bound_fortran() ==
-                "last_cell_all_colours_field1(colour)")
+                "last_edge_cell_all_colours_field1(colour)")
 
 
 def test_loop_start_expr(dist_mem):
     ''' Test that the start_expr property returns the expected reference
-    to a symbol. '''
+    to a symbol.
```

Comment:
I would suggest single quotes around `'start_expr'` in the line above.

Reply:
done